node-wasm/src/utils.rs

```diff
@@ -91,15 +86,4 @@ where
     {
         self.map_err(|e| JsError::new(&format!("{context}: {e}")))
     }
-
-    fn with_js_context<F, C>(self, context_fn: F) -> Result<T, JsError>
-    where
-        C: fmt::Display + Send + Sync + 'static,
-        F: FnOnce() -> C,
-    {
-        self.map_err(|e| {
-            let context = context_fn();
-            JsError::new(&format!("{context}: {e}"))
-        })
-    }
```

Comment:
What was the issue here?

Reply:
Unused

Reply:
I can keep it, but thought that we can easily add it when we need it